docs/chapters/flow/index.md

```diff
@@ -446,12 +446,36 @@ with:
 In this case, this construction lets the `FaissIndexer` use the `vectors` stored by the indexer named `wrapidx`. 
 
 
-### Using Queryset
+## Override parameters using QuerySet
 
-We can override parameter values in flows with the help of `queryset`. 
+We can override parameter values in flows with the help of `QuerySet`. `querySet` is a set of `QueryLang` protobuf messages that 
+can be sent along with any `Request`. It is useful to dynamically override parameters of a driver for a specific request. (Not every parameter
+is able to be overriden) 
```

Comment:
There is some unexpected line break here.

Reply:
Can u fix it in ur PR?